### PR TITLE
270 Bug fixes

### PR DESCRIPTION
**Pull request by sjpadgett**

### CODE REVIEW

File: sql/5_0_1-to-5_0_2_upgrade.sql [hunk 1]
```diff
@@ -853,14 +853,18 @@ DROP TABLE `eligibility_response`;
...
+#IfMissingColumn insurance_companies x12_default_eligibility_id
 ALTER TABLE `insurance_companies` ADD `x12_default_eligibility_id` INT(11)  DEFAULT NULL;
 #EndIf
 
 #IfMissingColumn users_secure login_fail_counter
 ALTER TABLE `users_secure` ADD `login_fail_counter` INT(11) DEFAULT '0';
 #EndIf
+
+#IfMissingColumn x12_partners x12_dtp03
+ALTER TABLE `x12_partners` ADD `x12_dtp03` CHAR(1) DEFAULT 'A';
```
Comment (bradymiller) [inline sql/5_0_1-to-5_0_2_upgrade.sql]: Also need to add change to database.sql

**Comment by bradymiller** [comment]
hi @sjpadgett , Only noted 1 issue. Otherwise code looks good to me :)

File: sql/5_0_1-to-5_0_2_upgrade.sql (hunk 1 above)
Comment (sjpadgett) [reply sql/5_0_1-to-5_0_2_upgrade.sql]: don't know what's wrong with me. very forgetful of late! resolved

Comment (sjpadgett) [reply sql/5_0_1-to-5_0_2_upgrade.sql]: lol, resolved the issue that is. forgetfulness still an issue though!

**Comment by sjpadgett** [comment]
@juggernautsei Note I put back the weno npi validation to 10 and fixed ncpdp to 7 from 10.
bringing in soon if nobody has anything....

**Comment by juggernautsei** [comment]
Thanks Jerry!

**Comment by sjpadgett** [comment]
@stephenwaite Bringing in. let me know if you still have issues and i'll take another wack

**Comment by stephenwaite** [comment]
test batch came out nice and was accepted by VT medicaid, should have results in a couple of hours :smile:

**Comment by stephenwaite** [comment]
results look great, thank you @sjpadgett

**Comment by sjpadgett** [comment]
Glad to hear. Another interop down. Now if I could get labcorp/quest to certify OpenEMR, we'd be golden. 

Which service date option did you wind up using in partner; default (appt date), current or effective?

**Comment by stephenwaite** [comment]
the default worked for my test site :) (funny how the github emoji smile is a little cray :smile: )

**Pull request merged**